src/components/dashboard/SendLinkSummary.js

```diff
@@ -251,9 +323,15 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
             <Section.Text color="gray80Percent" fontSize={14} style={styles.credsLabel}>
               To
             </Section.Text>
-            <Section.Text fontSize={24} fontWeight="medium" lineHeight={24} style={styles.toText}>
-              {counterPartyDisplayName}
```

Comment:
we should display "who". we can display both. first who and then address.

Reply:
The design sais to show just an address on the confirmation screen.